code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -35,12 +35,52 @@
 
 /mob/living/carbon/human/Move(NewLoc, direct)
 	. = ..()
-	if(shoes && .) // did we actually move?
+	if(.) // did we actually move?
 		if(!lying && !buckled)
 			if(!has_gravity(loc))
 				return
 			var/obj/item/clothing/shoes/S = shoes

```

Comment:
having the sanity check was causing an obscene amount of runtimes, should i be doing something diffrent?